src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Grammar.Test/tests/__snapshots__/GrammarTests.test.ts.snap

```diff
@@ -3670,29 +3777,305 @@ Line:
  - token from 0 to 1 () with scopes text.aspnetcorerazor
 
 Line:     if (true) {
```

Comment:
This `if (true) {` was in an existing test and we now properly detect what type of C# is being used in Razor code blocks.